doc/cephfs/cephfs-mirroring.rst

```diff
@@ -4,7 +4,7 @@
 CephFS Snapshot Mirroring
 =========================
 
-CephFS supports asynchronous replication of snapshots to a remote CephFS file system via
+CephFS supports asynchronous push based replication of snapshots to a remote CephFS file system via
```

Comment:
I would instinctively hyphenate push-based but will defer to @zdover23 

on line 9 below, does the parenthetical add value?

Reply:
@anthonyeleven Updated the PR. The parenthetical seems ok to me.